.github/workflows/ubuntu_install.yml

```diff
@@ -70,17 +70,23 @@ jobs:
       uses: mxschmitt/action-tmate@v3
       timeout-minutes: 15
       if: ${{ github.event_name == 'workflow_dispatch' && inputs.debug_enabled }}
+    - name: Run demos
+      run: |
+        source ~/openquake/bin/activate
+        oq dbserver start
+        cd ~/work/oq-engine/oq-engine
+        OQ_DISTRIBUTE=zmq bin/run-demos.sh demos/
     - name: Run tests
       run: |
         source ~/openquake/bin/activate
         pip install pytest
         oq dbserver start
         cd ~/work/oq-engine/oq-engine
-        oq engine --run demos/risk/ClassicalDamage/job_hazard.ini
         #
         export MPLBACKEND=Agg
-        pytest -vsx --color=yes openquake/hazardlib/tests/gsim/kotha_2020_test.py
-        pytest -vsx --color=yes openquake/sep/tests/test_utils.py
-        pytest -vsx --color=yes openquake/sep
-        #
+        pytest -vs --color=yes openquake/hazardlib/tests/gsim/kotha_2020_test.py openquake/sep/tests/test_utils.py openquake/sep
+    - name: Run risk_test
+      run: |
+        source ~/openquake/bin/activate
+        oq dbserver start
         oq engine --run https://downloads.openquake.org/jobs/risk_test.zip
```

Comment:
What are these tests?

Reply:
These tests run every night and they test running a calculation via https.